grid/recruitment/candidates/profile.supr

```diff
@@ -3,6 +3,7 @@ Manage Candidates
 Fetch, list and create candidates in ATS (Applicant Tracking System).
 
 @deprecated 
+This profile is deprecated and will be removed in the future. 
```

Comment:
@martinalbert this PR is based on wrong branch as it contains not relevant changes.

Reply:
Base branch is correct, only added this line because I add it also to `recruitment/jobs` profile description.

Reply:
I'd like to merge this PR and then merge main into #322.